### DISCUSSION TITLE
Hello. Is the prefetcher (Pythia Bingo SPP MLOP SMS) in the paper responsible for prefetching data from main memory to the cache hierarchy?

### DISCUSSION THREAD

**Discussion by szuhzw** [CMU-SAFARI/Hermes#27] [Q&A] [answered]
Hello. Is the prefetcher (Pythia Bingo SPP MLOP SMS) in the paper responsible for prefetching data from main memory to the cache hierarchy?
![AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA](https://github.com/CMU-SAFARI/Hermes/assets/65290731/448dc3fd-1591-48ee-a60e-7dc4a950f518)

**Comment by rahulbera** [comment, marked as answer]
Hi @szuhzw, yes. All prefetchers considered in this work (and this figure) is configured to prefetch data from the main memory up to LLC. The prefetcher codes are parameterized, so you can also easily configure them to prefetch to any cache level higher than LLC also.
